sips/sip-019/sip-019-modify-coinbase-reward-per-block-and-halving-schedule.md

```diff
@@ -0,0 +1,74 @@
+
+# Introduction
+Open membership blockchain based networks need to provide an economic incentive for miners to process transactions and provide security. Coinbase rewards are typically used to bootstrap this economic incentive in the early days of an open source network when transaction fees alone will not provide a sufficient incentive. Currently, the coinbase reward for the Stacks network is 1000 STX per block, with a halving schedule as follows:
+
+<img width="659" alt="Screenshot 2022-05-16 at 2 35 35 PM" src="https://user-images.githubusercontent.com/31934541/168593660-d13311d9-2710-4202-bc9d-e41bf4d6c945.png">
+
+For the first ten thousand blocks of Stacks 2.0, however, the coinbase reward was inflated to 2000 STX per block, and resulted in much more favorable STX mining and Stacking conditions. Since the purpose of a coinbase reward is to bootstrap the network in the early days, we propose increasing the coinbase reward and subsequently modifying the halving schedule.
+
+# Modifications
+This SIP proposes a change to the current STX coinbase reward, and therefore also a subsequent change to the halving schedule for the STX coinbase reward. The changes are as follows:
+
+## STX Coinbase Reward
+Change the current coinbase reward to 1600 STX per block. This change will remain in effect until the next halving for the Stacks network, which will occur roughly two years from the time of writing this draft.
+
+## STX Halving Schedule
+The current halving schedule for coinbase reward amounts are:
+- 1000 STX per block are released in the first 4 years of mining
+- 500 STX per block are released during the following 4 years
+- 250 STX per block are released during the following 4 years
+- 125 STX per block are released from then on indefinitely.
+
+This proposal would modify the schedule to the following:
+- 1600 STX per block are released from SIP acceptance until first halving (~2 years)
+- 800 STX per block are released during the following 4 years
+- 400 STX per block are released during the following 4 years
+- 200 STX per block are released during the following 4 years
+- 100 STX per block are released from then on indefinitely
+
+# Effects on STX Inflation & 2050 Supply
+The current cumulative mining mint results in a total circulating supply of approximately 1,810,745,848 STX tokens by January 2050. With the adoption of this proposal, the 2050 total circulating supply will increase by approximately 157 million STX tokens, to 1,967,777,098. Below is a visual comparison of the change.
+
+<img width="639" alt="Screenshot 2022-05-16 at 2 32 16 PM" src="https://user-images.githubusercontent.com/31934541/168593836-985e5613-80c8-42b0-8649-d3e6f9d595a1.png">
+
+<img width="641" alt="Screenshot 2022-05-16 at 2 25 14 PM" src="https://user-images.githubusercontent.com/31934541/168593867-f42714a0-4549-40e4-a0ad-fa15266ed269.png">
+
```

Comment:
Do you think you could include a graph of both the current and proposed change together, so the differences over time can be better appreciated?  Also, the table below is really helpful for inflation, but could you also include numbers for the absolute number of miner-released STX over time?  Thanks!